modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -1507,6 +1507,7 @@ export class Wallet implements IWallet {
       reshare: params.reshare,
       message: params.message,
       disableEmail: params.disableEmail,
+      skipKeychain: (this.type() ?? 'hot') === 'hot' ? !!params.skipKeychain : undefined,
```

Comment:
Given how sensitive wallet sharing is, I want users to explicitly pass this field in as input instead of deriving it based on other input params. This is why the params are passed as is for hot wallets instead of derived from other params such as `options.keychain`.